components/director/internal/statusupdate/repo_test.go

```diff
@@ -0,0 +1,175 @@
+package statusupdate_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/DATA-DOG/go-sqlmock"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/kyma-incubator/compass/components/director/internal/repo/testdb"
+	"github.com/kyma-incubator/compass/components/director/internal/statusupdate"
+	"github.com/kyma-incubator/compass/components/director/pkg/persistence"
+	"github.com/stretchr/testify/require"
+)
+
+const testID = "foo"
+
+func TestRepository_IsConnected(t *testing.T) {
+	testError := errors.New("test")
+
+	t.Run("Success for applications", func(t *testing.T) {
+
+		//GIVEN
+		db, dbMock := testdb.MockDatabase(t)
+		defer dbMock.AssertExpectations(t)
+		dbMock.ExpectQuery(regexp.QuoteMeta(`SELECT 1 FROM public.applications WHERE id = $1 AND status_condition = 'CONNECTED'`)).
+			WithArgs(testID).
+			WillReturnRows(testdb.RowWhenObjectExist())
+		ctx := persistence.SaveToContext(context.TODO(), db)
+		repo := statusupdate.NewRepository()
+
+		//WHEN
+		res, err := repo.IsConnected(ctx, testID, "applications")
+
+		//THEN
+		require.NoError(t, err)
+		assert.True(t, res)
+	})
+
+	t.Run("Success for runtimes", func(t *testing.T) {
```

Comment:
Do we need cases for `runtimes`?
How the differ from cases for `applications`?

The same question for `Update Query`